src/meta/src/manager/notification.rs

```diff
@@ -242,4 +249,27 @@ impl NotificationManagerCore {
 
         self.current_version
     }
+
+    #[expect(clippy::unused_async)]
+    async fn notify_all(&mut self, operation: Operation, info: &Info) -> NotificationVersion {
```

Comment:
How about make it non-async directly?

Reply:
yes, i will remove async and the clippy tag